src/rules/__tests__/utils.test.ts

```diff
@@ -18,7 +18,6 @@ const rule = createRule({
   name: __filename,
   meta: {
     docs: {
-      category: 'Possible Errors',
```

Comment:
These are picked up by tooling targeting v6 and v7, no?

Reply:
They're removed completely from `@typescript-eslint/eslint-plugin` and it's types, so we should remove them too.

Reply:
That's not what I asked 😅 What `@typescript-eslint/eslint-plugin` does is secondary, I care about other tooling. `@typescript-eslint` removing it from types is easy enough to work around if there is a non-zero value of keeping it around